websites/docs/src/content/docs/en/getting_started/installation.mdx

```diff
@@ -41,6 +54,69 @@ Before you can install and use Ballerine, you will need the following:
 
 After performing these steps, the backoffice should be running on http://localhost:5137/, the collection flow on http://localhost:5201/ and the workflow service will be accepting calls at http://localhost:3000/.
 
+### Installation steps on a remote virtual machine
+
+We recommend installation using ansible
+
+
+1. **Install ansible**: We recommend installing ansible with apt:
+<CodeBlock lang={`shell`}  code={`apt install -y ansible`}/>
+
+2. **Clone the project**: Use Git to clone the Ballerine repository to your local machine:
+<CodeBlock lang={`shell`}  code={`git clone https://github.com/ballerine-io/ballerine.git && cd ballerine`}/>
+
+3. **Switch to the dev branch**: After cloning, switch to the development branch:
+<CodeBlock lang={`shell`}  code={`git checkout dev`}/>
+
+4. **Navigate to ballerine_playbook directory**:
+<CodeBlock lang={`shell`}  code={`cd deploy/ansible/ballerine_playbook`}/>
+
+5. **Create inventory file**:
+<CodeBlock lang={`shell`}  code={`touch inventory.txt`}/>
+
+Now, with your editor, open the file and add the hostname or FQDN of the server(s) you want to deploy Ballerine to with the following pattern:
+
+6. **Add entries into the inventory file**:
+<CodeBlock lang={`shell`}  code={`all ansible_host={{ SERVER_HOST }} ansible_port={{ SERVER_PORT }} ansible_user={{ SERVER_USER }}`}/>
+
+If you are using SSH keypairs for authenticating your SSH connections to your server. You can tell Ansible your ssh private key file in the `inventory` file
+using `ansible_ssh_private_key_file`
+
+<CodeBlock lang={`shell`}  code={`all ansible_host={{ SERVER_HOST }} ansible_port={{ SERVER_PORT }} ansible_user={{ SERVER_USER }} ansible_ssh_private_key_file={{ SSH_PRIVATE_KEY_FILE }}`}/>
+
+After you completed the above step then we're pretty much done with the inventory
+
+7. **Setup your configuration vars for Ballerine**:
+
+The next step is to setup necessary configuration for your app to run such as environment variable, domain name, etc.
+
+First you need to open `deploy/ansible/ballerine_playbook/roles/setup-ballerine/defaults/main.yml` file with your editor.
+There are some variables that will need input from you to get the application start correctly
+
+- `install_dir`: The absolute path of your app's installation folder on the server (required). Default: `/home/ubuntu/ballerine`
```

Comment:
Is the default `ubuntu` or the user in `/home/ubuntu/ballerine`?

Reply:
Considering a linux machine we have gone by that. But users can changes it and we have mentioned it in the above line 
[here](https://github.com/ballerine-io/ballerine/pull/2177/files#diff-59d650ebcaf91827daff10112863f6671bdf436e809d47b2fc3ce8a3d7c7d44bR94)